### ISSUE TITLE
zoomTo() API is not defined

### ISSUE THREAD

**Issue by zcorpan** [Maps4HTML/HTML-Map-Element#41]
> The location and zoom level of the map created with the map element can be changed via the zoomTo(...) API method, which will change the zoom, lat and lon properties.

This API is not defined. Does it change the zoom, lat, and lon content attributes?

**Comment by prushforth** [comment]
> This API is not defined. Does it change the zoom, lat, and lon content attributes?

Yes, that's the [notion](https://maps4html.org/HTML-Map-Element/spec/#the-map-element).  

I've encountered a [problem](https://github.com/google/WebFundamentals/issues/8309) in setting the checked attribute boolean attributes based on an API in custom elements, because it seems to set up an infinite loop, where if you're watching the attribute(s), you then kick off a property change, which then changes the attribute and so on.  I just haven't had time to [go back](https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/66) and get it correctly worked out and implemented.
